test/parsertests.js

```diff
@@ -5,6 +5,7 @@
  */
 
 var runCommon = function(testcases) {
+    xpathmodels = XPathModels();
```

Comment:
What hashtag config does this use? If not the default, then it's not obvious what this is doing. Would it make sense to setup the default/global `xpathmodels` this way and have the first line in the `XPathModels` function be `hashtagConfig = hashtagConfig || defaultHashtagConfig;`?